importer/importer.py

```diff
@@ -166,6 +176,11 @@ def get_items(connection,
         class_to_use = SPECIFIC_TABLES[specific_table_name]["class"]
```

Comment:
I would add `specific_table = `SPECIFIC_TABLES[specific_table_name]` here so that you don't need the full path later.

You can also drop the `.keys()` on the line above.